src/cpuid/src/cpu_leaf.rs

```diff
@@ -61,6 +61,8 @@ pub mod leaf_0x1 {
     }
 
     pub mod edx {
+        pub const MCE_BITINDEX: u32 = 7; // Memory Check Exception
```

Comment:
Since we added these definitions, it would be good to have the bits set in the templates as well since that was the intention.

Reply:
done